src/main/java/com/denizenscript/depenizen/bukkit/utilities/mythicmobs/MythicMobsLoaders.java

```diff
@@ -22,8 +22,12 @@ public void RegisterEvents() {
     // @group Depenizen Bridges
     // @plugin Depenizen, MythicMobs
     // @description
-    // In addition to the the tags, commands, and events found by searching for "mythicmobs" throughout the meta documentation,
-    // Depenizen adds additional features to Mythic Mobs: 2 Targeters, and a Condition.
+    // In addition to the tags, commands, and events found by searching for "mythicmobs" throughout the meta documentation, 4 new Denizen matchers are added:
+    // - "mythic_mob" plaintext EntityTag matcher, matches if the entity is a mythic mob.
+    // - "mythic_mob:<MythicMobID>" EntityTag matcher, matches if the entity is a mythic mob, and it's ID matches the advanced matcher specified.
+    // - "mythic_item" plaintext ItemTag matcher, matches if the item is a mythic item.
+    // - "mythic_item:<MythicItemID>" ItemTag matcher, matches if the item is a mythic item, and it's ID matches the advanced matcher specified.
+    // Depenizen also adds additional features to Mythic Mobs: 2 Targeters, and a Condition.
```

Comment:
Should put the core list of things all in one block, and then the details of the new matchers in its own block, similar to the pre-existing organization of this meta.